packages/mapviewer/src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -1,59 +1,87 @@
 <script setup>
 import DOMPurify from 'dompurify'
 import log from 'geoadmin/log'
-import { computed } from 'vue'
+import { computed, toRefs } from 'vue'
 import { useI18n } from 'vue-i18n'
 import { useStore } from 'vuex'
 
 import SelectableFeature from '@/api/features/SelectableFeature.class.js'
-import { WHITELISTED_HOSTNAMES } from '@/config/staging.config'
+import { BLOCKED_EXTENSIONS, WHITELISTED_HOSTNAMES } from '@/config/staging.config'
 import FeatureAreaInfo from '@/modules/infobox/components/FeatureAreaInfo.vue'
 import FeatureDetailDisclaimer from '@/modules/infobox/components/FeatureDetailDisclaimer.vue'
 import CoordinateCopySlot from '@/utils/components/CoordinateCopySlot.vue'
 import allFormats from '@/utils/coordinates/coordinateFormat'
 
-const { feature } = defineProps({
+const props = defineProps({
     feature: {
         type: SelectableFeature,
         required: true,
     },
 })
 
+const { feature } = toRefs(props)
+
 const { t } = useI18n()
 
 const store = useStore()
-const hasFeatureStringData = computed(() => typeof feature?.data === 'string')
-const popupDataCanBeTrusted = computed(() => feature.popupDataCanBeTrusted)
+const hasFeatureStringData = computed(() => typeof feature.value?.data === 'string')
+const popupDataCanBeTrusted = computed(() => feature.value.popupDataCanBeTrusted)
```

Comment:
you can remove the `.value` accessor to all `feature` call after you've removed the props `toRefs`